### ISSUE TITLE
Task 1: Multi-task setting for training of stage 1

### ISSUE THREAD

**Issue by eeishaan** [eeishaan/pretrain-gnns#1]
1. The first stage is a single task setting now. We want to convert it to multi-task setting. 
1. Train the 1st stage with Making and ContextPred as target.
1. Possible changes include: loss function modification, data loader modification, network modification. Please note that this is not an exhaustive list.
